### DISCUSSION TITLE
@Space()이 vsCode에서 적용이 안되는 것 같습니다.

### DISCUSSION THREAD

**Discussion by zlmrzlo** [naverz/zepeto-studio-kor#1249] [5. Q&A]
한 번 사용해보려고 시도를 해봤는데  안되는 것 같네요.

혹시 다른 분들은 되시나요?

**Comment by YSHFM** [comment]
![image](https://user-images.githubusercontent.com/106574541/188102174-1f71c176-6c29-495b-8e2f-1a142c8a4ad0.png)
![image](https://user-images.githubusercontent.com/106574541/188102211-5da244ba-c28b-42b3-bc23-a32e69a8ab22.png)
잘 적용 되네요

**Reply by zlmrzlo** [reply]
@header는 잘 적용되고 @Space는 적용이 안되어서 뭔가 문제 있다고 생각했는데 다시 한 번 확인해봐야 할 것 같네요 ㅠㅠ
